src/components/atoms/ChatMessage/ChatMessage.tsx

```diff
@@ -36,7 +38,9 @@ export const ChatMessage: React.FC<ChatProps> = ({
 
   return (
     <div className={containerStyles}>
-      <div className="chat-message__text">{text}</div>
+      <div className="chat-message__text">
+        <Emoji text={text} />
+      </div>
```

Comment:
As per https://github.com/sparkletown/sparkle/pull/1394#issuecomment-841597607, pending discussion in https://github.com/sparkletown/sparkle/issues/1393#issuecomment-841595445 to see if we want to just use `emoji-mart`, which has its own `Emoji` component for rendering emoji:

- https://github.com/missive/emoji-mart#emoji